Makefile

```diff
@@ -8,7 +8,7 @@ install:
 
 # run all tests, generate allure report
 tests: .dirs
-	pytest cardano_node_tests -n 8 --artifacts-base-dir=.artifacts/ --cli-coverage-dir=.cli_coverage/ --alluredir=.reports/
+	pytest cardano_node_tests -n auto --artifacts-base-dir=.artifacts/ --cli-coverage-dir=.cli_coverage/ --alluredir=.reports/
```

Comment:
Here the number of processes can be higher than number of available cores/threads, because most of the time tests are just waiting for next epoch etc., not really creating any load on CPU.